tests/RootFile/testCleantalkSFWUpdateDownload.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Cleantalk\ApbctWP\Helper;
+
+class SfwUpdateDownloadTest extends TestCase
+{
+    private $apbctBackup;
+    private $helper;
+
+    protected function setUp(): void
+    {
+        global $apbct;
+
+        $this->apbctBackup = $apbct;
+
+        $apbct = new \Cleantalk\ApbctWP\State('cleantalk', array('settings', 'data', 'errors', 'remote_calls', 'stats', 'fw_stats'));
+
+        $apbct->api_key = getenv("CLEANTALK_TEST_API_KEY");
```

Comment:
Can we not use an API key in tests?

Reply:
You are right, removed it https://github.com/CleanTalk/wordpress-antispam/pull/709/commits/ce9ea8aacab8360e803df105f1ab272dd297691e